tests/__tests__/sitka_module.test.ts

```diff
@@ -86,13 +89,39 @@ describe("SitkaModule", () => {
     expect(currentState).toEqual(defaultTextModuleState)
   })
 
-//   // SUBSCRIPTION
-//   test('subscriptions are created/provided with provideSubscriptions & createSubscription', () => {
-//   })
+  // SUBSCRIPTION
+  test('subscriptions are created/provided with provideSubscriptions & createSubscription', () => {
+    const { sitka, store } = createSitkaAndStore()
+    const { text: textModule, color: colorModule } = sitka.getModules()
+    // Validates that we start with default state
+    const startingEditsState = textModule.getStateTestDelegate(store.getState()).numberOfEdits
+    expect(startingEditsState).toEqual(0)
+    // Validates that subscribed function is called and updates state
+    colorModule.handleColor("blue")
+    const updatedEditsState = textModule.getStateTestDelegate(store.getState()).numberOfEdits
```

Comment:
how do i know that handleColor blue isn't also changing the numberOfEdits using mergeState?

Reply:
Okay good point instead I set up a spy